user-guide/Troubleshooting/Procedures/Troubleshooting_Cassandra.md

```diff
@@ -0,0 +1,157 @@
+---
+uid: Troubleshooting_Cassandra
+---
+
+# Troubleshooting – Cassandra
+
+## Context
+
+DataMiner uses Cassandra in various ways. Each way has its specific behavior and place within a DataMiner System.
+
+Two types of Cassandra storage are used, i.e. two distinct types of Cassandra databases:
+
+- Dedicated clustered storage:, called *Cassandra Cluster* below.
+- Storage per DMA (with or without indexing), called *Cassandra Single* below.
+
+> [!TIP]
+> For more information on available storage options, see [About databases](xref:Databases_about).
+
+When you start troubleshooting, you first need to determine whether the setup uses *Cassandra Cluster* or *Cassandra Single*. There are some key differences between the two that are important for your investigation.
```

Comment:
Can we add a title above this line, e.g. "Determining the type of setup"?

Reply:
I'm not sure, the goal of the titles was to keep them the same across all troubleshooting pages and I don't know if we will always have a "Determining the type of setup". 
Perhaps we can add it and if the troubleshooting item is generic across setups, this title section would simply highlight (avoid confusion) that it doesn't matter what type of setup you have.
So: yes, add the title and we can evaluate along the way.